src/azure-cli/azure/cli/command_modules/security/commands.py

```diff
@@ -285,6 +293,12 @@ def load_command_table(self, _):
         g.custom_command('list', 'list_security_sub_assessments')
         g.custom_command('show', 'get_security_sub_assessment')
 
+    with self.command_group('security adaptive_application_controls',
+                            security_adaptive_application_controls_sdk,
+                            client_factory=cf_security_adaptive_application_controls) as g:
+        g.custom_command('list', 'list_security_adaptive_application_controls')
+        g.custom_command('show', 'get_security_adaptive_application_controls')
```

Comment:
```suggestion
        g.custom_show_command('show', 'get_security_adaptive_application_controls')
```

Reply:
Why not use custom_show_command?

Reply:
Merge this time, and please fix all of it next time.